easybuild/easyconfigs/n/netMHC/netMHC-4.0a.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'Tarball'
+
+name = 'netMHC'
+version = '4.0a'
+
+homepage = 'http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?netMHC'
+description = """ NetMHC 4.0 software  predicts binding of peptides to a number of different
+ HLA alleles  using artificial neural networks (ANN). """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# netMHC is proprietary software, but free for academcs
+# see: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?netMHC
+source_urls = ['http://www.cbs.dtu.dk/services/NetMHC-4.0']
+sources = [
+    {
+        'filename': '%(name)s-%(version)s.Linux.tar.gz',
+        'extract_cmd': 'tar --strip-components=1 -xzf %s',
+    },
+    {
+        'download_filename': 'data.tar.gz',
+        'filename': '%(name)s-%(version)s-data.tar.gz',
+    },
+]
+checksums = [
+    'd9938388c50bf49d748e5c1293435d7d9fae5d0a33303d795e767429508e368a',  # netMHC-4.0a.Linux.tar.gz
```

Comment:
@smoors I'm getting a different SHA256 checksum for this source tarball...

```
f50b4eca8f8c5b9184c5a5be423e9806c54bd22ddeb4596f5c246597c59f1a34
```

Reply:
@boegel that's weird, I downloaded it again and the checksum is still the same.

Reply:
Just did, I get the same checksum as before. Maybe there's a fingerprint on the provided tarball?

Reply:
Nevermind, I was actually downloading `netMHCpan-4.0a.Linux.tar.gz` rather than `netMHC-4.0a.Linux.tar.gz`...

Software naming, it's a black art...